src/pyenphase/envoy.py

```diff
@@ -88,16 +96,20 @@ def __init__(
         host: str,
         client: httpx.AsyncClient | None = None,
         timeout: float | httpx.Timeout | None = None,
+        max_request_delay: int | None = None,
     ) -> None:
         """Initialize the Envoy class."""
         # We use our own httpx client session so we can disable SSL verification (Envoys use self-signed SSL certs)
         self._timeout = timeout or LOCAL_TIMEOUT
+        self._max_request_delay = max_request_delay or MAX_REQUEST_DELAY
```

Comment:
I think we can drop this and use the constant. I don't think this is a good use case for making it configurable

Reply:
I was thinking if the timeout is configurable this one should be too, as they have kind of a relation. But it seems not straight forward to implement it in this way.

Reply:
Removed the parameter.

Is 4 retries or 50 sec the right sweetspot for this? If we make it 4 and 65 sec there will be 1 retry on read timeout.

Reply:
I think we can lower the read timeout to 45s. I had it set to 60 because we had trouble with the HA event loop being blocked but thats since been fixed

Reply:
> I was thinking if the timeout is configurable this one should be too, as they have kind of a relation. But it seems not straight forward to implement it in this way.

I'm not sure we ever pass in a timeout and it probably doesn't make much sense to allow setting one (but it would be a breaking change to remove)

Reply:
Still find it appealing to have both configurable as I've seen my part of slow Envoy's that we got going by fine tuning on individual base. But I can live with it being static.